_currencies.md

```diff
@@ -1,5 +1,80 @@
 
-In our Knowledge Base you will always find the most recent and up-to-date [list of the currencies we support](https://support.uphold.com/hc/en-us/articles/202473803), along with the three letter abbreviation you can use when referring to the currency within the API.
+<aside class="notice">
+  For brevity and convenience, we refer to any such asset simply as "currency" throughout the API documentation.
+  In cases that only apply to specific classes of assets, we will clarify accordingly.
+</aside>
+
+In Uphold's API, various endpoints include currencies in their input or output. We represent all such currencies by an abbreviation **code** of variable length, typically containing uppercase letters (for example, a currency's [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) code, an equity's [ticker symbol](https://en.wikipedia.org/wiki/Ticker_symbol), or other similar well-known representation).
+
+## List Supported Assets
+
+> Example of unauthenticated request to list all supported assets:
+
+```bash
+curl https://api.uphold.com/v0/assets
+```
+
+> The above command returns JSON in the following format:
+
+```json
+[
+  {
+    "code": "BTC",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 8
+    },
+    "name": "Bitcoin",
+    "status": "open",
+    "symbol": "฿",
+    "type": "cryptocurrency"
+  },
+  {
+    "code": "USD",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 2
+    },
+    "name": "US Dollar",
+    "status": "open",
+    "symbol": "$",
+    "type": "fiat"
+  }
+]
+```
+
+> Example of authenticated request to list only the assets available to the current user:
+
+```bash
+curl https://api.uphold.com/v0/assets \
+  -H "Authorization: Bearer <token>"
+```
+
+Get the list of supported currencies and other financial assets, with details on their name, type, [status](#asset-status), and formatting specification for user-facing display.
+
+If the request is unauthenticated, the full list of assets supported by Uphold is returned.
+Authenticated requests, on the other hand, will filter the output, returning only the assets available for the current user, which can depend on factors such as their country and state.
+
+Please note that this endpoint is [paginated](#pagination), so multiple requests may be required to get the complete list.
+
+### Asset status
+
+The `status` field of an asset indicates whether it can be transacted at the moment.
+The possible values are:
+
+* `open` — the asset is in regular operation and transactions can be made with it.
+* `closed` — the asset is temporarily disabled;
+  used for equities during periods outside the U.S. stock market's trading hours
```

Comment:
missing the comma?

Reply:
After "equities"? If so, I didn't feel it was necessary, but no objections to adding it. Please confirm.

Reply:
No, after hours.

Reply:
I'm pretty sure it's not correct to use a comma before an opening parenthesis. Maybe you misinterpreted the first `(` in the `([...](...) ...)` of the line below as part of the markdown link syntax?

Reply:
Yeah, too many brackets, I assumed they were all functional :stuck_out_tongue:

Reply:
I thought there was a [relevant xkcd](https://xkcd.com/297), but the one I had in mind is more appropriate :P

<img src="https://live.staticflickr.com/2182/2065970378_7b03194210_c.jpg" width="400"/>

(p.s. — I am not a Lisper 😅)